app.js

```diff
@@ -331,7 +331,9 @@ class FilterByTags {
 
   updateTagButtons() {
     // Update the appearance of tag buttons based on the selected state
-    const tagButtons = document.querySelectorAll(".filter__selection__tags__button");
+    const tagButtons = document.querySelectorAll(
+      ".filter__selection__tags__button"
+    );
```

Comment:
This code only changes the formatting of this line of code, and there are other changes in this PR that only affects whitespace (the "spaces").

Having such changes in a git diff makes it much more difficult for a reviewer to sort out the actually meaningful changes to the code, increasing the mental load of the reviewer, and hence the risk that they miss something important.

It's a good idea at the beginning of any team project to agree on rules for formatting (e.g. quote marks, indentation etc) and configure editors or other automated software to enforce that formatting. That way, no one will ever commit code that doesn't conform with the rules, and no one will have to change formatting as part of their PRs.